README.md

```diff
@@ -32,6 +32,7 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 | [Redlight Software](https://weareredlight.com) | Web and mobile development studio focused on products. | `Coimbra` `Lisboa` |
 | [Subvisual](https://subvisual.co/) [:rocket:](https://subvisual.co/company/join-us/) | Web development and design firm. | `Braga` |
 | [Thing Pink](http://www.thing-pink.pt) [:rocket:](http://www.thing-pink.pt/joinus) | Digital Agency with focus on Web, Mobile and IoT. | `Lisboa` `Porto` |
+| [TMC](https://tmc-employeneurship.com/) [:rocket:](https://www.linkedin.com/company/tmcspain/jobs/) | Agile development for Aeronautic, FIN(TECH) and insurance customers. | `Porto` |
```

Comment:
Can you please shorten a little bit the description?  
I want to keep the horizontal space under control.

Reply:
Yes. I need to fix my repo so I need to create another PR.